MBINCompiler/Models/NMSTemplate.cs

```diff
@@ -126,6 +126,17 @@ public static NMSTemplate DeserializeBinaryTemplate(BinaryReader reader, string
                         }
 
                         break;
+                    case "Object":

```

Comment:
@naomichan
Right, I missed that NMSTemplate is not abstract and it's already used in templates [like this](https://github.com/emoose/MBINCompiler/blob/976a4ebf22dce9711638a60fd7172e92e23cee27/MBINCompiler/Models/Structs/GcHUDLayerData.cs#L8).
Simply creating a "NMSTemplate"-case to read the concrete type won't work because of that.